ios/MarkdownLayoutManager.mm

```diff
@@ -25,9 +27,13 @@ - (void)drawBackgroundForGlyphRange:(NSRange)glyphsToShow atPoint:(CGPoint)origi
       CGFloat y = paddingTop + rect.origin.y;
       CGFloat width = markdownUtils.markdownStyle.blockquoteBorderWidth;
       CGFloat height = rect.size.height;
-      CGRect lineRect = CGRectMake(x, y, width, height);
-      [markdownUtils.markdownStyle.blockquoteBorderColor setFill];
-      UIRectFill(lineRect);
+      CGFloat nestShift = paddingLeft + width + markdownUtils.markdownStyle.blockquoteMarginLeft;
+
+      for(int strip = 0; strip < currentDepth; strip++) {
+        CGRect lineRect = CGRectMake(x + (strip * nestShift), y, width, height);
```

Comment:
The idea behind these variables (`x`, `y`, `width`, `height`) was that they are calculated above and thus we don't need any calculations in `CGRectMake`. Please move the calculations outside of it.
```suggestion
        CGRect lineRect = CGRectMake(x, y, width, height);
```